completions/zsh/_sway

```diff
@@ -1,17 +1,5 @@
-#compdef sway
-#------------
-# Description
-# -----------
-#
-# Completion script for the sway window manager (http://swaywm.org)
-#
-# ---------------------------------------------
-# Author
-# -------
-#
-# * Seth Barberee <seth.barberee@gmail.com>
-#
-# -------------------------------
```

Comment:
We should keep these, wouldn't be nice to remove the cred from Seth :)

Reply:
Woops. I meant to get rid of the unnecessary comments. I'll keep the author mention.